.travis.yml

```diff
@@ -2,8 +2,11 @@ language: python
 python:
   - 3.6
 install:
+  - pip install -r pipenv.txt
   - pip install flake8
```

Comment:
We can remove this line in a followup commit, as we're now running `flake8` via `pipenv`